docs/installation.rst

```diff
@@ -12,9 +12,9 @@ Prerequisites
 .. _driver link: https://www.nvidia.com/drivers
 
 1. Linux x86_64
-2. `CUDA 12.1 <https://developer.nvidia.com/cuda-downloads>`__ or later.
-3. |driver link|_ supporting CUDA 12.1 or later.
-4. `cuDNN 8.9 <https://developer.nvidia.com/cudnn>`__ or later.
+2. `CUDA 11.8 <https://developer.nvidia.com/cuda-downloads>`__
+3. |driver link|_ supporting CUDA 11.8 or later.
+4. For FP8 fused attention, `CUDA 12.1 <https://developer.nvidia.com/cuda-downloads>`__ or later, |driver link|_ supporting CUDA 12.1 or later, and `cuDNN 8.9 <https://developer.nvidia.com/cudnn>`__ or later.
```

Comment:
Why are we reverting to listing CUDA 12.1 and cuDNN 8.9 as optional dependencies? They are currently required. Making them optional would be great in a follow-up PR though.

Reply:
Currently, the code can compile with lower versions and at runtime, it will error out when FP8 APIs are called, so it's better to keep the installation requirement low and ask them to upgrade to higher versions if they need FP8. Otherwise, everyone will have to upgrade.

Reply:
Fixed installation.rst to reflect the current requirement for cuDNN in compilation of TE. Will remove this requirement in the future for non-fused-attention use cases.